tutorials/English/1.Getting_Started.ipynb

```diff
@@ -65,15 +43,13 @@
    },
    "outputs": [],
    "source": [
-    "# Clone the repo\n",
+    "# Install OmniSafe and dependencies from source\n",
+    "## if you already have omnisafe installed, please ignore the code in this cell.\n",
```

Comment:
It would better to unify ``omnisafe`` and ``OmniSafe``

Reply:
Accept it!